src/http/httpd.cc

```diff
@@ -121,7 +128,7 @@ future<> connection::start_response() {
         });
     }
     set_headers(*_resp);
-    _resp->_headers["Content-Length"] = to_sstring(
+    _resp->_headers["Content-Length"] = size_to_sstring(
```

Comment:
instead turning away from `to_sstring()` and creating `size_to_sstring()` as a more performant alternative, IMHO, i think a better way to address this issue is to fix `to_sstring()`.

see #1398

Reply:
I'm not sure how `fmt` works. From https://github.com/scylladb/seastar/pull/1398:

```
template <typename string_type, typename T>
string_type to_sstring(T value) {
    auto size = fmt::formatted_size("{}", value);
    auto formatted = uninitialized_string<string_type>(size);
    fmt::format_to(std::back_inserter(formatted), "{}", value);
    return formatted;
```

Will this really compile to something as fast as `size_to_sstring`?

Reply:
Well, apparently it should be in the ballpark: https://www.zverovich.net/2020/06/13/fast-int-to-string-revisited.html

Reply:
Alright, I will close the PR for now. Maybe I'll compare the two when #1398 is merged.